src/Ens/JobeetBundle/DataFixtures/ORM/LoadJobData.php

```diff
@@ -0,0 +1,55 @@
+// src/Ens/JobeetBundle/DataFixtures/ORM/LoadJobData.php
```

Comment:
@PhotonSapiens 
You missed `<?php` at the beginning - so file is interpreted as plain text, not code